Updates - Read Below

MapGuy11 opened the pull request:
1. Change MeTV West (I assume) to MeTV Arizona from (Distributed Antenna System)
2. Add Pennsylvania Cable Network from (BozzTV/TeleupTV)
3. Add Entertainment Studios Channels from (BozzTV/TeleupTV)
4. Add Popular Digital Subchannels from (Distributed Antenna System)
5. Add 2 **NEW** Subchannels in the US _True Real_ and _Defy TV_

Closes https://github.com/iptv-org/iptv/issues/3733

MapGuy11 committed:
Add PCN

MapGuy11 committed:
Update us.m3u

jkemp814 committed:
Matching call signs with EPG

MapGuy11 committed:
Merge branch 'master' into master

Dum4G commented on channels/us.m3u:
```diff
@@ -6558,9 +6558,39 @@ http://188.40.68.167/russia/disney/playlist.m3u8
\ No newline at end of file
+https://streams.the6tv.duckdns.org:2443/locals/Pheonix/kazt-dt5.m3u8
+#EXTINF:-1 tvg-id="PCN.us" tvg-name="Pennsylvania Cable Network" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/11044/s11044_h3_aa.png" group-title="News",Pennsylvania Cable Network
+https://bozztv.com/teleyupp/teleup-pcnhd/playlist.m3u8
```
Every BozzTV link is Geo-Blocked

MapGuy11 replied:
Will Fix

Dum4G commented on channels/us.m3u:
```diff
@@ -6558,9 +6558,39 @@ http://188.40.68.167/russia/disney/playlist.m3u8
+https://streams.the6tv.duckdns.org:2443/locals/Pheonix/kazt-dt5.m3u8
+#EXTINF:-1 tvg-id="PCN.us" tvg-name="Pennsylvania Cable Network" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/11044/s11044_h3_aa.png" group-title="News",Pennsylvania Cable Network
+https://bozztv.com/teleyupp/teleup-pcnhd/playlist.m3u8
+#EXTINF:-1 tvg-id="CARSTV.us" tvg-name="Cars.TV" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/71302/s71302_h3_aa.png" group-title="Entertainment",Cars.TV
```
Sounds like Auto category

MapGuy11 replied:
Will Fix

jkemp814 committed:
Matching call signs for EPG , names, logos

iptv-bot[bot] committed:
[Bot] Update playlists (#3736)

Co-authored-by: freearhey <freearhey@users.noreply.github.com>

jkemp814 committed:
Matched call sign with EPG - Change name

MapGuy11 committed:
Changes Below

1. Add [Geo-blocked] to all BozzTV excluding no TeleupTV
2. Remove Duplicate Entries
3. Teleup Changed from WCTX to MyTV closest to their Cable Provider WHVL-LD1
4.

MapGuy11 committed:
Merge branch 'master' of https://github.com/MapGuy11/iptv

jkemp814 committed:
Merge branch 'master' into channel-edits-0

jkemp814 committed:
Rename CMT Pluto to Pluto TV CMT, changed logo

Dum4G commented:
I'll check every BozzTV channel for geo-block and resolve merge conflict first. Channels you have added didn't open for me but KTLA5 works just fine

Dum4G commented:
@LaneSh4d0w  I'v got an interesting behavior to check. I was able to watch every BozzTV channel with teleyupp in links just fine, but when I tried one with teleyupp1 it started to restrict me. Please check if you can reproduce the same

LaneSh4d0w commented:
> @LaneSh4d0w  I'v got an interesting behavior to check. I was able to watch every BozzTV channel with teleyupp in links just fine, but when I tried one with teleyupp1 it started to restrict me everywhere. Please check if you can reproduce the same
> 
> UPD: as expected, it was a temporal block

I'll check that, had a few things to do.

jkemp814 committed:
SportsGrid - Rename, change logos, sync tags

jkemp814 committed:
Shop HQ - Rename, change logo, sync tags

jkemp814 committed:
Revry News - Added missing logo, sync tags

jkemp814 committed:
RiffTrax - Corrected caps in name, added language

Dum4G committed:
Update rs.m3u

jkemp814 committed:
Merge pull request #3735 from jkemp814/channel-edits-0

Matching call sign with EPG

jkemp814 committed:
Merge branch 'iptv-org:master' into channel-edits-1

Dum4G committed:
Merge branch 'master' into Dum4G-patch-1

jkemp814 committed:
STIRR News - Added missing logo

jkemp814 committed:
Surf Now TV - Syncronized tags

jkemp814 committed:
Tennis Channel - Changed name, sync logos

Dum4G committed:
Update si.m3u

Dum4G committed:
Update ba.m3u

jkemp814 committed:
Toon Goggles - Corrected name, sync tags

Dum4G committed:
Update hr.m3u

jkemp814 committed:
The Young Turks (TYT) - Rename, sync tags

Dum4G committed:
Update hr.m3u

jkemp814 committed:
WMFD-TV 68 Mansfield - Changed name

jkemp814 committed:
FOX and ABC - Renamed remaining few news channels

jkemp814 committed:
ABC 7 Washington D.C. (WJLA) - Rename, sync tags

jkemp814 committed:
COZI TV (KNSD-DT2) - Changed tvg-id to match EPG

jkemp814 committed:
My 11 Monterey - Changed name, sync tags

jkemp814 committed:
ABC7 News San Francisco (KGO) - Renamed, Sync tags

jkemp814 committed:
CBC News - Corrected logo, displaying CBS

jkemp814 committed:
CBSN - Rename channels, fix call sign (reverted)

jkemp814 committed:
FOX - Fixed description to match name

jkemp814 committed:
News channel edits - Renaming with call sign

jkemp814 committed:
myTV 4 San Francisco Bay Area (KRON-TV) - Renamed

jkemp814 committed:
NBC 4 News Washington DC (WRC-TV) - Renamed

jkemp814 committed:
KRCB - Rename channels

jkemp814 committed:
PBS - Fixed comma between city and state

LaneSh4d0w commented:
Links I've checked worked fine, no blockage whatsoever @Dum4G

Dum4G committed:
Update ba.m3u

Dum4G commented:
> Links I've checked worked fine, no blockage whatsoever @Dum4G

Even if you try teleyupp1 link and then smth else?

Dum4G committed:
Merge pull request #3739 from iptv-org/Dum4G-patch-1

Ex-Yu: cleanup and channel additions

LaneSh4d0w commented:
Yes, actually i've alternated between teleyupp and teleyupp1, then going back to teleyupp, and all of them worked fine (tested C-Span, Fox 7 and Freeform)

Dum4G committed:
Merge branch 'master' into channel-edits-1

Dum4G commented:
I'm removing geo-blocked tag then

jkemp814 committed:
Fixed tvg-id tag, Changed name on FOX affiliate

jkemp814 committed:
Merge branch 'channel-edits-1' of https://github.com/jkemp814/iptv into channel-edits-1

jkemp814 committed:
KTVK-DT5 - Rename channel

jkemp814 committed:
KVVB-TV 33 Victor Valley - Fixed tvg-id tag

jkemp814 committed:
ABC 3.5 Arizona Weather - Fixed group title

Dum4G committed:
Merge pull request #3738 from jkemp814/channel-edits-1

Channel edits continue: Syncing tags, matching logos

Dum4G committed:
Merge branch 'pr/3734'

Dum4G committed:
Update us.m3u

Dum4G commented:
At this point I don't want to expirement with Github Desktop merge implementation. I'll do all of the changes from here in a new PR
